verticaltabs.js

```diff
@@ -488,6 +490,11 @@ VerticalTabs.prototype = {
         `
     });
 
+    pin_button.addEventListener('click', function () {
+      //ss is undefined within the other click handler
```

Comment:
It is?  Why?  Oooh, cause the other one is running in a string?  If that's the case, then let's pull all that code out of the string, and put it in this function.